### DISCUSSION TITLE
Naming: finalize what we call a physical cluster

### DISCUSSION THREAD

**Discussion by ncdc** [kcp-dev/kcp#1930] [General]
Let's make our naming consistent for "physical clusters" in docs, help text, etc. Some options:

- physical cluster
- actual cluster
- compute cluster
- compute target
- real cluster

any other ideas?

**Comment by ncdc** [comment]
* workload cluster

**Comment by stevekuznetsov** [comment]
Workload cluster is a good one I think

**Comment by stevekuznetsov** [comment]
@sttts @davidfestal @jmprusi @qiujian16 @s-urbaniak @p0lyn0mial @astefanutti WDYT? :)

**Comment by stevekuznetsov** [comment]
Also @MikeSpreitzer @csams @kylape @nrb @clubanderson

**Comment by nrb** [comment]
👍 to workload cluster

**Comment by p0lyn0mial** [comment]
I also like workload cluster

**Comment by clubanderson** [comment]
works for me
